<launch>
   <node pkg="rbx2_diagnostics" type="monitor_laptop_charge.py" name="monitor_laptop_charge" output="screen">
      <param name="warn_percent" value="50" />
      <param name="error_percent" value="20" />
   </node>
   
   <node pkg="diagnostic_aggregator" type="aggregator_node" name="diagnostic_aggregator" clear_params="true">
      <rosparam command="load" file="$(find rbx2_diagnostics)/config/power.yaml" />
   </node>

   <node pkg="rqt_robot_monitor" type="rqt_robot_monitor" name="rqt_robot_monitor" />
</launch>